### PR TITLE
[Merged by Bors] -  Don't run init container as root and avoid chmod and chowning

### PR DESCRIPTION
**Pull request by sbernauer**
## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Comment by sbernauer** [comment]
Started test https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/opa-operator-it-custom/22/

**Comment by sbernauer** [comment]
Tests passed

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ All notable changes to this project will be documented in this file.
 
 - Updated stackable image versions ([#374]).
 - `operator-rs` `0.22.0` -> `0.27.1` ([#377]).
+- Don't run init container as root and avoid chmod and chowning ([#381]).
 
 [#374]: https://github.com/stackabletech/opa-operator/pull/374
 [#377]: https://github.com/stackabletech/opa-operator/pull/377
+[#381]: https://github.com/stackabletech/opa-operator/pull/381
```
Comment (maltesander) [inline CHANGELOG.md:15]:
```suggestion
- Don't run init container as root and avoid chmod and chowning ([#382]).

[#374]: https://github.com/stackabletech/opa-operator/pull/374
[#377]: https://github.com/stackabletech/opa-operator/pull/377
[#382]: https://github.com/stackabletech/opa-operator/pull/382
```

**Review by maltesander — APPROVED** [review]
LGTM if pr number fixed!

**Pull request closed without merging**